### ISSUE TITLE
New maintainer proposal

### ISSUE THREAD

**Issue by marcospb19** [gustavla/timeit#6]
Hey @gustavla.

I like `timeit`, although it is a small crate, I think there are a couple of things that could improve, PRs are stale right now so I assume you're either busy or not interested in maintaining it anymore.

Would you be willing to hand it off to someone else? If you're OK with it, I'd like to maintain timeit, WDYT?

**Comment by vext01** [comment]
I was just looking for something maintained like timeit. There haven't been commits in over 8 years.

I guess you could just fork?

**Comment by marcospb19** [comment]
@vext01 Indeed, 8 years is a lot, I'll think about creating my own crate @gustavla doesn't answer this one. 

That said, I prefer to ask and wait for a bit before taking action, I'm expecting to come to a better solution together.

It all depends on how much Gustav is willing to hand it off (and trust my judgment/experience/skills :smile: ).
